src/proto/Solana.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package TW.Solana.Proto;
+option java_package = "wallet.core.jni.proto";
+
+message Transfer {
+    bytes private_key = 1;
+    string recipient = 2;
+    uint64 amount = 3;
+}
+
+message Stake {
+    bytes from_private_key = 1;
+    bytes stake_private_key = 2;
+    string vote_pubkey = 3;
+    uint64 amount = 4;
```

Comment:
if you are using decimals `13`, then I would suggest to use `BigInt` or bytes for amount, to avoid overflow